<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HumbleMaps</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Users/Алексей/Desktop/icon100x100.png</normaloff>../Users/Алексей/Desktop/icon100x100.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_street">
         <property name="title">
          <string>Свойства улицы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>улица</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_street">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_node">
         <property name="title">
          <string>Свойства перекрёстка</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>карта</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>правка</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>вид</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>инструменты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>справка</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMap"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionContents">
   <property name="text">
    <string>содерджание</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>о программе</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>actionOpen</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>редактировать улицы</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>редактировать дорожные покрытия</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>редактировать автомобили</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>редактировать водителей</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>редактирование видов топлива</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
